head.html

```diff
@@ -221,33 +248,79 @@
         links.forEach((url) => {
           url = url.trim();
           if (!url) return;
-          const { title, type } = getSocialLinkDetails(url);
+          const { title, type, className } = getSocialLinkDetails(url);
           const link = document.createElement('a');
-          link.className = `social-${type}`;
+          link.className = className;
           link.title = title;
           link.href = url;
           link.appendChild(createSVG(type));
           socialWrap.appendChild(link);
         });
-        target.appendChild(socialWrap);
+        target.insertBefore(socialWrap, source.nextSibling);
       }
     }
   }
 
+  function fetchLatestPosts() {
+    const latestWrap = document.createElement('div');
+    latestWrap.className = 'default latest-posts';
+    getSection().parentNode.appendChild(latestWrap);
+    const searchClient = algoliasearch('LPQI0MG7ST', '9bf61456f606d21ddc1723f30500659e');
```

Comment:
Are we supposed to expose those tokens like this ?

Reply:
yes, these are fine: App ID and read only search token.